libs/filabridge/include/private/filament/UibStructs.h

```diff
@@ -151,12 +151,14 @@ struct alignas(256) PerRenderableUib {
     static constexpr utils::StaticString _name{ "ObjectUniforms" };
     math::mat4f worldFromModelMatrix;
     math::mat3f worldFromModelNormalMatrix;   // this gets expanded to 48 bytes during the copy to the UBO
+    alignas(16) math::mat3f triplanarOrientationMatrix;   // this gets expanded to 48 bytes during the copy to the UBO
     alignas(16) math::float4 morphWeights;    // morph weights (we could easily have 8 using half)
     uint32_t flags;                           // see packFlags() below
     uint32_t channels;                        // 0x000000ll
     uint32_t objectId;                        // used for picking
     // TODO: We need a better solution, this currently holds the average local scale for the renderable
     float userData;
+    alignas(16) math::float3 worldAabbCenter; // center of renderable's AABB in world space
```

Comment:
I don't think it will be useful for goals in this form: renderables are part-level, but we need coherent texturing on the body-level. Adjacent parts would show a seam in texturing.

If it's indeed important to adjust triplanar center to the AABB center, it's likely the easiest to solve by some form of inheritance in the transform hierarchy. It's a bit tricky, since lower level entities have their own AABB, which we want to ignore. (I'd consider a new `float3` property, or maybe pack in material orientation matrix as a 4x4).

Reply:
That's actually a good question design-wise too: if these are two different materials, do we really _need_ them to be seamless? Ideally, this should be situational (and toggle-able). This should be also discussed with Alex.

Reply:
Added support for custom material orientation center in 1f9fca5